.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+on:
+  push:
+    tags: '*'
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  release:
+    runs-on: windows-2019 # Code signing requirement https://github.com/NuGet/Home/issues/7939
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Parse repo name
+        run: |
+          $FullName = "$env:GITHUB_REPOSITORY"
+          $Org,$RepoName = $FullName.Split('/')
+          echo "PARTICULAR_REPO_NAME=$RepoName" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        shell: pwsh
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: 5.0.x
+      - name: Setup NuGet for signing
+        uses: nuget/setup-nuget@v1.0.5
+        with:
+          nuget-api-key: ${{ secrets.NuGetAPIKey }}
```

Comment:
Is this the name of the org level secret? If so, it should probably be `NUGET_API_KEY` to match the convention of the existing ones.